### DISCUSSION TITLE
Calling fzf_configure_bindings once

### DISCUSSION THREAD

**Discussion by luong-komorebi** [PatrickF1/fzf.fish#207] [Ideas]
Firstly, thank you for the awesome plugins.

I am new to fish shell, so this question might be silly but do we have a way to call fzf_configure_bindings once? This makes sense on the UX side because I do not change my keybindings too often that much and I would like the configs to be persistent - in a way that it is set up once and forever.

Also, looking at https://github.com/PatrickF1/fzf.fish/blob/a07e175e5b9c581d042c96cbd0bea8296a0d34f8/functions/fzf_configure_bindings.fish#L20-L24 makes me wonder if we can turn these into universal variables so the performance of the shell improves and we dont have to call `fzf_configure_bindings` every time we start up the shell

**Comment by PatrickF1** [comment]
Hi Luong! Thank you for your kind words.

> do we have a way to call fzf_configure_bindings once

You mean besides adding it to config.fish? There isn't. I could potentially use universal variables but that affords me less flexibility with future changes to `fzf_configure_bindings` and also universal variables are hard to work with and they might be going away in a far future fish release.
Btw, adding configuration to config.fish is not unusual. Plenty of other plugins do it too.

**Reply by luong-komorebi** [reply]
Thank you for your prompt reply @PatrickF1 . Just to clarify, I feel good putting it in config.fish and no I am not asking for another use case. I just wanna hint at performance optimisation, since if we are allowed to set it up once and forget about it, we wont have to re-call fzf_configure_bindings every time we start the shell. My shell is fast, just curious about the possibility since the use case for me is valid and I dont usually change fzf key bindings :D

**Comment by kidonng** [comment]
You can make `fzf_configure_bindings` an empty function ("no-op") and customize the bindings yourself. That said, the performance trade-off of those `set` commands is too tiny to matter, you don't need to worry about a thing.

**Reply by luong-komorebi** [reply]
thank you @kidonng. Maybe I was up for a premature optimisation :D
